python/lsst/validate/drp/base.py

```diff
@@ -26,3 +26,7 @@
 class ValidateError(Exception):
     """Base classes for exceptions in validate_drp."""
     pass
+
+class ValidateErrorNoStars(ValidateError):
+    """Base classes for exceptions in validate_drp."""
```

Comment:
(I don't see how to comment on the general comment).  The way that emacs identifies files is suffix, but failing that a comment of the form `-*- python -*-`, not the `#!` line